src/lib/gridsterCompact.service.ts

```diff
@@ -47,7 +47,6 @@ export class GridsterCompact {
     if (widgetMovedUp) {
```

Comment:
@tiberiuzuld : Just asking, shouldn't we add else for the **if statements** like this?